moped-editor/src/views/projects/projectsListView/components/saveViewModal.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { useMutation } from "@apollo/client";
+import { Dialog, DialogTitle, DialogContent, IconButton } from "@mui/material";
+import CloseIcon from "@mui/icons-material/Close";
+import SaveViewForm from "./saveViewForm";
+import { useLocation } from "react-router-dom";
+import { ADD_USER_SAVED_VIEW } from "src/queries/project";
+
+const SaveViewModal = ({
+  showDialog,
+  setIsSaveViewModalOpen,
+  filters,
+  filtersLabels,
+  setIsViewSaved,
+  handleSnackbar,
+}) => {
+  const [saveView] = useMutation(ADD_USER_SAVED_VIEW);
+
+  let { pathname, search } = useLocation();
+
+  const defaultDescription = filtersLabels
+    .map(
+      (filter) =>
+        `${filter.filterLabel} ${filter.operatorLabel} ${filter.filterValue}`
+    )
+    .join(", ");
+
+  const onClose = () => {
+    setIsSaveViewModalOpen(false);
+  };
+
+  // run save view mutation
+  const onSaveViewClick = () => {
+    saveView({
+      variables: {
+        object: {
+          description: defaultDescription,
```

Comment:
could you update this to save from the form data that is passed from React Hook Form? I noticed that if I try to overwrite the default description that the view still saves with the default description. Thanks!

Reply:
sorry i missed that!